### PR TITLE
update document of basic

### PR DESCRIPTION
**Pull request by csvwolf**
Please makes sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you follow Element's [Contributing Guide](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.md).
* [x] Rebase before creating a PR to keep commit history clear.
* [x] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

File: examples/docs/message.md [hunk 1]
```diff
@@ -95,7 +95,7 @@
 
 ### 不同状态
 
-Message 提供了四种类型：`success`，`info`，`warning`，`error`，由不同图标表示。
+Message 提供了四种类型： `success` ， `info` ， `warning` ， `error` ，由不同图标表示。
```
Comment (Leopoldthecoder) [inline examples/docs/message.md]: 中文逗号的两侧似乎不需要空格？

**Comment by QingWei-Li** [comment]
lgtm

File: examples/docs/message.md (hunk 1 above)
Comment (QingWei-Li) [inline examples/docs/message.md]: 是的

Comment (csvwolf) [inline examples/docs/message.md]: @Leopoldthecoder @QingWei-Li 😯改了

**Pull request merged**